### ISSUE TITLE
Home Mortgage Disclosure Act - research data

### ISSUE THREAD

**Issue by NealHumphrey** [codefordc/housing-insights#130]
Research what data is available via the Home Mortgage Disclosure Act. This has been recommended by a couple of people that we have interviewed or that have heard about the project. 

- How granular can we get data (city level, census tract level, etc.)
- What data fields are available? What caveats are there to them? What do they tell us about the city?
- What time periods is the data available for?
- What data format is it provided in? 
- Where are updates to the data posted, and how frequently?
- What fields are available to connect this data to our other data sets? e.g. zip code, census tract, etc. Do we need to create any sort of cross-walk table to translate between them?

As soon as the data ingestion pipeline is set up, we can also download and integrate them into the current system (can make a separate issue for that when the time comes).

**Comment by NealHumphrey** [comment]
@emkap01 This might be a good next task for you to look into while we wait to decide what to do about the PUDs! Let me know if you want to grab it (you can assign yourself using assignees if so)

**Comment by emkap01** [comment]
I grabbed it. I'll start looking into it.

**Comment by emkap01** [comment]
I've started a [draft report](https://docs.google.com/document/d/1xcPpX37nq4TUpb-YNcDM0U915z0Eq7DkALohKOPJeG8/edit?usp=sharing) which includes a link to a sample of the data.  I'll continue updating it as I learn more. Based on what I've seen so far, it looks like the HMDA records could be useful for establishing mortgage trends at the census tract level.  There were ca. 35,000 HMDA records for DC in 2015.  It's available via a well-documented API and ostensibly includes records back to 2007.

**Comment by NealHumphrey** [comment]
@emkap01 cool, good stuff. I watched the video. So it looks like each record is an individual loan application (accepted or denied)? Did you get a sense of whether the data is complete (i.e. all loans in the US) or if there's any particular bias to which types of banks submit to HMDA and which types of loans they submit?

**Comment by emkap01** [comment]
Yes, each record represents an individual loan application, regardless of whether it was accepted or denied.  The full dataset contains ca. 35 million records and ostensibly represents the entire U.S.  There is a field labeled 'purchaser_type_name' that indicates which type of entity bought the loan, however it only contains a single value to describe any type of bank - ' Commercial bank, savings bank or savings association' - so it looks like it's not necessarily possible to sub-categorize banks in this dataset.

In addition to loading the sample csv on Drive (which I did primarily because that makes it easier to point Tableau Public at it and build charts with it), I also pushed a copy to the 'raw' S3 bucket.  Bear in mind that this csv file represents a subset of the available data, and is specifically limited to HMDA records for DC from 2015.
